### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by shpraka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,14 +18,15 @@ NOTE: you must have write permissions to the repository in question. If you're u
...
+* [Checkout](https://github.com/actions/checkout) your Git repository content into GitHub Actions agent.
 * Authenticate using [Azure Web App Publish Profile](https://github.com/projectkudu/kudu/wiki/Deployment-credentials#site-credentials-aka-publish-profile-credentials) or using the [Azure Login Action](https://github.com/Azure/login). Examples of both are given later in this article.
 
     The action supports using publish profile for [Azure Web Apps](https://azure.microsoft.com/services/app-service/web/) (both Windows and Linux) and [Azure Web Apps for Containers](https://azure.microsoft.com/services/app-service/containers/) (both Windows and Linux).
     
  **Note: As of October 2020, Linux web apps will need the app setting `WEBSITE_WEBDEPLOY_USE_SCM` set to `true` before downloading the publish profile from the portal. This requirement will be removed in the future.**
 
     The action does not support multi-container scenario with publish profile.
+* Enable [Run from Package](https://docs.microsoft.com/en-us/azure/app-service/deploy-run-package#enable-running-from-package), otherwise remote build will take time and the deployment  will take longer.
```
Comment (kumaramit-msft) [inline README.md:29]: Is there extra space - `deployment  will` ?

File: README.md [hunk 2]
```diff
@@ -18,14 +18,15 @@ NOTE: you must have write permissions to the repository in question. If you're u
 
 ## Dependencies on other GitHub Actions
 
-* [Checkout](https://github.com/actions/checkout) Checkout your Git repository content into GitHub Actions agent.
```
Comment (kumaraksh1) [inline README.md:21]: I think this is correct. 
Github Action name and what it does.

Apart from Amit's comment on extra space, looks good to me.

File: README.md (hunk 1 above)
Comment (shpraka) [reply README.md]: Updated.

**Review by kumaramit-msft — APPROVED** [review]

**Review by kumaraksh1 — APPROVED** [review]
LGTM!!

**Pull request merged**